### ISSUE TITLE
ther is no base.h file 

### ISSUE THREAD

**Issue by Sandy4321** [blazs/HoeffdingTree#2]
visual 2015 give error 
Severity	Code	Description	Project	File	Line	Suppression State
Error	C1083	Cannot open include file: 'base.h': No such file or directory	HoeffdingTree	d:\a_start_oct30_2016\moa\hoeffdingtree-master-cpp\hoeffdingtree-master\hoeffding.h	4

pls share this file 	
 base.h

**Comment by blazs** [comment]
Yes, the dependency is the [GLib]() library developed by a research group. The library has been open-sourced, so I can push it to the repo over the weekend.

**Comment by blazs** [comment]
Dear Sandy, I will rewrite the code over the upcoming holidays. The idea is to get rid of all external dependencies and to write the code in proper modern C++. Let me know if you're still interested in using the tool.
